src/runtime/assemblymanager.cs

```diff
@@ -196,7 +196,17 @@ public static Assembly LoadAssembly(string name)
             Assembly assembly = null;
             try
             {
-                assembly = Assembly.Load(name);
+                var importEvent = new ImplicitAssemblyLoadingEventArgs(name);
```

Comment:
@dmitriyse why did you augment assembly loading with this event, I cannot see the purpose of this.